docs/SelectInput.md

```diff
@@ -108,16 +108,18 @@ const choices = [
 
 ## `emptyValue`
 
-An empty choice is always added (with a default `''` value, which you can overwrite with the `emptyValue` prop) on top of the options. You can furthermore customize the `MenuItem` for the empty choice by using the `emptyText` prop, which can receive either a string or a React Element, which doesn't receive any props.
+An empty choice is always added (with a default `''` value, which you can override with the `emptyValue` prop) on top of the options. You can furthermore customize the `MenuItem` for the empty choice by using the `emptyText` prop, which can receive either a string or a React Element, which doesn't receive any props.
 
 ```jsx
-<SelectInput source="category" emptyValue={null} choices={[
+<SelectInput source="category" emptyValue="" choices={[
```

Comment:
```suggestion
<SelectInput source="category" emptyValue={0} choices={[
```

You should choose an example with a value other than the default, otherwise it's not very demonstrative :sweat_smile: